build.gradle

```diff
@@ -8,14 +8,15 @@ group 'run.halo.oauth'
 sourceCompatibility = JavaVersion.VERSION_17
 
 repositories {
+    mavenLocal()
     maven { url 'https://s01.oss.sonatype.org/content/repositories/releases' }
     maven { url 'https://s01.oss.sonatype.org/content/repositories/snapshots/' }
     maven { url 'https://repo.spring.io/milestone' }
     mavenCentral()
 }
 
 dependencies {
-    implementation platform('run.halo.tools.platform:plugin:2.17.0-SNAPSHOT')
+    implementation platform('run.halo.tools.platform:plugin:2.20.0+local.5')
```

Comment:
Change the version to `2.20.0-SNAPSHOT` after merging <https://github.com/halo-dev/halo/pull/6702>.